src/PrestaShopBundle/Resources/config/admin/routing_product.yml

```diff
@@ -60,14 +60,6 @@ admin_product_form:
     requirements:
         id: \d+
 
-admin_product_use_legacy_setter:
```

Comment:
not this is not a BC, this code and the related code should be deleted before 1.7.0!